x-pack/plugin/old-lucene-versions/src/internalClusterTest/java/org/elasticsearch/xpack/lucene/bwc/AbstractArchiveTestCase.java

```diff
@@ -94,11 +94,7 @@ public IndexMetadata getSnapshotIndexMetaData(RepositoryData repositoryData, Sna
                             .put(
                                 IndexMetadata.SETTING_INDEX_VERSION_CREATED.getKey(),
                                 metadata.settings()
-                                    .getAsVersionId(
-                                        "version",
-                                        IndexVersion::fromId,
-                                        IndexVersion.fromId(randomFrom(5000099, 6000099, 7000099))
-                                    )
+                                    .getAsVersionId("version", IndexVersion::fromId, IndexVersion.fromId(randomFrom(5000099, 6000099)))
```

Comment:
version 7 here was added as part of the lucene 10 ugprade but it was premature, as archive indices did not support yet lucene 8 indices.